css/src/tailwind.js

```diff
@@ -77,6 +77,10 @@ config.borderWidths = Object.assign(config.borderWidths, {
   '3': '3px'
 })
 
+config.height = Object.assign(config.height,{
```

Comment:
Don't need this, it already exists https://github.com/citizennet/purescript-ocelot/pull/44/files#L24

Reply:
I meant to link here https://github.com/citizennet/purescript-ocelot/blob/master/css/src/tailwind.js#L24